Curve-finance-suite/DAOKillingGauge/src/agent.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { HandleTransaction } from "forta-agent";
+import { TestTransactionEvent, createAddress } from "forta-agent-tools";
+import { providerHandleTransaction, createFinding } from "./agent";
+import { gaugeInterface } from "./abi";
+
+const curveDAOAddress = createAddress("0x1");
+
+describe("Curve DAO Kill a Gauge Test Suite", () => {
+  let handleTransaction: HandleTransaction;
+
+  beforeAll(() => {
+    handleTransaction = providerHandleTransaction(curveDAOAddress);
+  });
+
+  it("should return empty findings method was not called", async () => {
+    const txEvent = new TestTransactionEvent();
+
+    const findings = await handleTransaction(txEvent);
+
+    expect(findings).toStrictEqual([]);
+  });
+
+  it("should return a finding method set_killed was called from DAO", async () => {
```

Comment:
same previous note, we can use:
Should return a finding when set_killed method is called from DAO